packages/venia-ui/lib/components/CartPage/cartPage.css

```diff
@@ -0,0 +1,76 @@
+.root {
+    padding: 1.5rem;
+}
+
+.body {
+    display: grid;
+    gap: 2rem;
+    grid-template-columns: 66% auto;
+    /* The summary grid item spans the entire right column. */
+    grid-template-areas:
+        'items summary'
+        'adjustments summary'
+        'recents summary';
+}
+
+.heading_container {
+    display: flex;
+    align-items: baseline;
+    justify-content: space-between;
+
+    margin-bottom: 1rem;
+}
+
+.heading {
+    line-height: 1.25em;
+}
+
+.items_container {
+    grid-area: items;
+}
+
+.price_adjustments_container {
+    grid-area: adjustments;
+}
+
+.recently_viewed_container {
+    grid-area: recents;
+}
+
+.sign_in {
+    font-weight: bold;
+    text-decoration: underline;
+}
+
+.summary_container {
+    grid-area: summary;
+    height: 100%;
+}
+
+.summary_contents {
+    position: sticky;
+    /*
+     * TODO: Use CSS Properties (variables) or something instead of hardcoding these. 
+     * min-height of nav header + page spacing + height of page header 
+     */
+    top: calc(3.5rem + 3.5rem + 25px);
```

Comment:
This is not sticking as I would expect. Add some extra long text like lorem ipsum and scroll. You'll see that it scrolls off the page.

Reply:
Oh yeah, that's what

> Scroll and verify that the "Order Summary" container "sticks" to the side / stays in view. Note that it is expected that it will disappear once you scroll past all of the content.

was supposed to identify 😛.

If we're talking about the same thing, the fact that you can scroll past the content entirely is a product of a [style that gets applied to all pages](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/lib/components/Main/main.css#L13): the initial height of the page content area is always set to `100vh`.

I think once the components on this page are present we won't be able to repro this.